Remotion/Data/DomainObjects.UnitTests/ClientTransactionFactoryTest.cs

```diff
@@ -54,10 +54,9 @@ public void CreateRootTransaction_WithExtension ()
           .Callback((ClientTransaction transaction) => transaction.Extensions.Add(extensionStub.Object))
           .Verifiable();
 
-      ITransaction transaction = factory.Object.CreateRootTransaction();
+      var transaction = factory.Object.CreateRootTransaction();
```

Comment:
This test verifies that the interface-version of the API works correctly. Therefor, we should keep the original test and instead do a cast:
((IClientTransactionFactory)factory.Object)